tests/rt.conf

```diff
@@ -85,21 +85,21 @@ RUN | cpld_debug_pdlib_p8                               | - noaacloud
 
 ### ATM tests ###
 COMPILE | atm_dyn32 | intel | -DAPP=ATM -DCCPP_SUITES=FV3_GFS_v16,FV3_GFS_v16_flake,FV3_GFS_v17_p8,FV3_GFS_v17_p8_rrtmgp,FV3_GFS_v15_thompson_mynn_lam3km,FV3_WoFS_v0,FV3_GFS_v17_p8_mynn,FV3_GFS_v17_p8_ugwpv1 -D32BIT=ON | | fv3 |
-RUN | control_flake                                     |                            | baseline |
+##RUN | control_flake                                     |                            | baseline |
 RUN | control_CubedSphereGrid                           |                            | baseline |
 RUN | control_CubedSphereGrid_parallel                  | - noaacloud                | baseline |
 RUN | control_latlon                                    |                            | baseline |
 RUN | control_wrtGauss_netcdf_parallel                  |                            | baseline |
 RUN | control_c48                                       |                            | baseline |
-RUN | control_c48.v2.sfc                                |                            | baseline |
+##RUN | control_c48.v2.sfc                                |                            | baseline |
 RUN | control_c192                                      | - noaacloud                | baseline |
 RUN | control_c384                                      |                            | baseline |
-RUN | control_c384gdas                                  | - noaacloud                | baseline |
-RUN | control_stochy                                    |                            | baseline |
-RUN | control_stochy_restart                            | - noaacloud                |          | control_stochy
-RUN | control_lndp                                      |                            | baseline |
-RUN | control_iovr4                                     |                            | baseline |
-RUN | control_iovr5                                     |                            | baseline |
+##RUN | control_c384gdas                                  | - noaacloud                | baseline |
+##RUN | control_stochy                                    |                            | baseline |
+##RUN | control_stochy_restart                            | - noaacloud                |          | control_stochy
+##RUN | control_lndp                                      |                            | baseline |
+##RUN | control_iovr4                                     |                            | baseline |
+##RUN | control_iovr5                                     |                            | baseline |
```

Comment:
Why are these tests commented out?

Reply:
I went ahead and kept the tests running as is since they are being worked on for compatibility with v17. Anning recommended that control_csawmgt and control_csawmgt_debug be removed, and that has also been done in the latest version of this PR.

We can go in later and replace the individual tests as the v17 updates are completed.